.github/workflows/test-snap-can-build.yml

```diff
@@ -11,16 +11,44 @@ permissions:
 jobs:
   build:
     runs-on: ubuntu-latest
-
     steps:
       - uses: actions/checkout@v3
 
-      - uses: snapcore/action-build@v1
+      - name: Setup snapcraft
+        env:
+          LP_BUILD_SECRET: ${{ secrets.LP_BUILD_SECRET }}
```

Comment:
Apparently, we can't actually use secrets in PR builds: https://securitylab.github.com/research/github-actions-preventing-pwn-requests/

Reply:
I think the best approach might be to create two PR workflows:

* The original one that builds inside of github and uses `on: pull_request`. This will only test the amd64 build.
* A new one that uses `on: pull_request_trigger`, but only triggers when the submitter is part of snapcrafters (has `write` access on the repo). This new one uses the `remote-build` functionality.

This way, the launchpad build secrets are only exposed to snapcrafters' code. Feel free to change the PR workflow back to the original one, and create the second workflow at a later time, so we can start testing the current code.